### ISSUE TITLE
RL Group Meeting

### ISSUE THREAD

**Issue by NirViaje** [NirViaje/nirviaje.github.io#72]
-

**Comment by NirViaje** [comment]
# 1st meeting

- Overview
- MBRL with Learned model
- on skills transfer and more
  -

**Comment by lucasshen123** [comment]
# Current Session

## Window 1

* [SemanticKITTI - A Dataset for LiDAR-based Semantic Scene Understanding](http://www.semantic-kitti.org/dataset.html#download)
* [New Tab](chrome://newtab/)
* [Downloads](chrome://downloads/)
* [python -> - Google 搜索](https://www.google.com.hk/search?q=python+-%3E&oq=python+-%3E&aqs=chrome..69i57j69i65j69i60.3648j0j7&sourceid=chrome&ie=UTF-8)
* [segmentfault.com](https://segmentfault.com/q/1010000009112777)
* [​Python 3 新特性：类型注解 - 知乎](https://zhuanlan.zhihu.com/p/37239021)
* [blog.csdn.net](https://blog.csdn.net/sunt2018/article/details/83022493)
* [procgen - Google 搜索](https://www.google.com.hk/search?q=procgen&oq=procgen&aqs=chrome..69i57j0l9.2192j0j7&sourceid=chrome&ie=UTF-8)
* [Procgen Benchmark](https://openai.com/blog/procgen-benchmark/)
* [def __call__ - Google 搜索](https://www.google.com.hk/search?q=def+__call__&newwindow=1&safe=strict&ei=bvdaYJehNZGS0PEPgaiDuAM&oq=def+__call__&gs_lcp=Cgdnd3Mtd2l6EAwyAggAMgIIADIECAAQHjIECAAQHjIECAAQHjIECAAQHjIECAAQHjIECAAQHjIGCAAQChAeMgYIABAKEB5Q0tEhWNLRIWDs2SFoAHAAeACAAc8CiAHVBJIBBTItMS4xmAEAoAECoAEBqgEHZ3dzLXdpesABAQ&sclient=gws-wiz&ved=0ahUKEwiXkJ7QwMjvAhURCTQIHQHUADcQ4dUDCA0)
* [Python中的__init__()和__call__()函数_正西风落叶下长安-CSDN博客](https://blog.csdn.net/Yaokai_AssultMaster/article/details/70256621)
* [Python super() 函数 | 菜鸟教程](https://www.runoob.com/python/python-func-super.html)
* [Python 单例模式讲解和代码示例](https://refactoringguru.cn/design-patterns/singleton/python/example)
* [Python单例模式(Singleton)的N种实现 - 知乎](https://zhuanlan.zhihu.com/p/37534850)
* [.initialize(batch_size)_百度搜索](https://www.baidu.com/s?ie=utf-8&f=8&rsv_bp=1&tn=baidu&wd=.initialize(batch_size)&oq=kwargs%255B%2526%252339%253Bproposals%2526%252339%253B%255D&rsv_pq=d4c7bb8600146d1a&rsv_t=c8b0eaJVlk4X5i6dagkg1xgkQd9y3ReZpdlxAFZJ1UhICghwStR7jzWITeY&rqlang=cn&rsv_dl=tb&rsv_enter=0&rsv_btype=t&inputT=2507&rsv_n=2&rsv_sug3=17&rsv_sug4=2507)
* [（九）深度学习实战 | MMDetection之FCOS（2）_Skies_的博客-CSDN博客](https://blog.csdn.net/Skies_/article/details/109956705)
* [键值对_dai_wen的博客-CSDN博客_键值对](https://blog.csdn.net/dai_wen/article/details/79187591)
* [Python中的*args和**kwargs(python基础学习)_loner_fang的博客-CSDN博客](https://blog.csdn.net/loner_fang/article/details/80652161)
* [blog.csdn.net](https://blog.csdn.net/weixin_39825045/article/details/111803603)
* [（九）深度学习实战 | MMDetection之FCOS（2）_Skies_的博客-CSDN博客](https://blog.csdn.net/Skies_/article/details/109956705)
* [gin-gonic/gin: Gin is a HTTP web framework written in Go (Golang). It features a Martini-like API with much better performance -- up to 40 times faster. If you need smashing performance, get yourself some Gin.](https://github.com/gin-gonic/gin)
* [www.google.com](https://www.google.com/search?q=npz&oq=npz&aqs=chrome..69i57j69i64j69i60l2.4047j0j7&sourceid=chrome&ie=UTF-8)
* [tf.split 与 tf.squeeze 用法 - 知乎](https://zhuanlan.zhihu.com/p/52087724)
* [blog.csdn.net](https://blog.csdn.net/leviopku/article/details/82380118)
* [使用 tf.distribute.Strategy 进行自定义训练  |  TensorFlow Core](https://www.tensorflow.org/tutorials/distribute/custom_training?hl=zh-cn)
* [www.zybuluo.com](https://www.zybuluo.com/Team/note/1491361)
* [TensorFlow 2.0 下多 GPU 自定义并行训练 - 知乎](https://zhuanlan.zhihu.com/p/88165283)
* [TensorFlow 2.0 简明指南 - 知乎](https://zhuanlan.zhihu.com/p/70232196)
* [Tensorflow上手4: 初探分布式训练. 今天打算简单讲讲Tensorflow分布式训练的方法和遇到的一些问题。感兴趣的朋… | by Dong Wang | Medium](https://yaoyaowd.medium.com/tensorflow%E4%B8%8A%E6%89%8B4-%E5%88%9D%E6%8E%A2%E5%88%86%E5%B8%83%E5%BC%8F%E8%AE%AD%E7%BB%83-12a5e2b2db77)
* [tensorflow/mirrored_strategy.py at master · tensorflow/tensorflow](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/distribute/mirrored_strategy.py#L455)
* [horizon reinforcement learning - 搜索结果 - 知乎](https://www.zhihu.com/search?type=content&q=horizon+reinforcement+learning)
* [MirroredStrategy - 搜索结果 - 知乎](https://www.zhihu.com/search?type=content&q=MirroredStrategy)
* [【分布式训练】单机多卡的正确打开方式（二）：TensorFlow | 瓦特兰蒂斯](https://fyubang.com/2019/07/14/distributed-training2/)
* [python 3.x - Multi-CPU usage in tf.distribute.MirroredStrategy() -> tensorflow 2.1.0 - Stack Overflow](https://stackoverflow.com/questions/65065113/multi-cpu-usage-in-tf-distribute-mirroredstrategy-tensorflow-2-1-0)
* [Python 按键(key)或值(value)对字典进行排序 | 菜鸟教程](https://www.runoob.com/python3/python-sort-dictionaries-by-key-or-value.html)
* [排序指南 — Python 3.9.2 文档](https://docs.python.org/zh-cn/3/howto/sorting.html)
* [https://www.tensorflow.org/guide/distributed_training](https://www.tensorflow.org/guide/distributed_training)
* [https://www.tensorflow.org/tutorials/distribute/custom_training](https://www.tensorflow.org/tutorials/distribute/custom_training)
* [https://www.tensorflow.org/guide/distributed_training?hl=zh-cn](https://www.tensorflow.org/guide/distributed_training?hl=zh-cn)
* [tensorflow多机/多GPU训练时使用的术语_Daniel2333的博客-CSDN博客](https://blog.csdn.net/weixin_35653315/article/details/71403562)
* [github.com](https://github.com/tensorflow/tensorflow/issues/32010)
* [kwargs - Doge 搜索](https://www.dogedoge.com/results?q=kwargs)
* [1. *args and **kwargs — Python Tips 0.1 documentation](https://book.pythontips.com/en/latest/args_and_kwargs.html)
* [Python tips: 什么是*args和**kwargs？ - MK2 - 博客园](https://www.cnblogs.com/fengmk2/archive/2008/04/21/1163766.html)
* [GRU - 搜索结果 - 知乎](https://www.zhihu.com/search?q=GRU&type=content)
* [long short term memory - Doge 搜索](https://www.dogedoge.com/results?q=long+short+term+memory)
* [Long short-term memory - Wikipedia](https://en.wikipedia.org/wiki/Long_short-term_memory)
* [nc.dvi](https://www.bioinf.jku.at/publications/older/2604.pdf)
* [[译] 理解 LSTM(Long Short-Term Memory, LSTM) 网络 - wangduo - 博客园](https://www.cnblogs.com/wangduo/p/6773601.html)
* [bdd100k - Doge 搜索](https://www.dogedoge.com/results?q=bdd100k)
* [www.jiqizhixin.com](https://www.jiqizhixin.com/articles/UC-Berkeley-BDD100K)
* [LSTM IIR - Doge 搜索](https://www.dogedoge.com/results?q=LSTM+IIR)
* [为什么很少拿神经网络来直接做滤波器呢？ - 知乎](https://www.zhihu.com/question/264043819)
* [CLINK: Compact LSTM Inference Kernel for Energy Efficient Neurofeedback Devices](https://vast.cs.ucla.edu/sites/default/files/publications/CLINK_ISLPED%202018%20publication.pdf)
* [FSMN论文阅读笔记 - 知乎](https://zhuanlan.zhihu.com/p/348953069)
* [Lecture 27: Recurrent Neural Nets](http://www.isle.illinois.edu/speech_web_lg/coursematerials/ece417/18fall/ece417fa2018lec27.pdf)
* [Recurrent Networks](https://www.inf.ed.ac.uk/teaching/courses/mlp/2015/mlp09-rnn.pdf)
* [Reinforcement Learning, 3D Understanding, Robotics · Issue #33 · ExponentialDeepSpace/eds-archive](https://github.com/ExponentialDeepSpace/eds-archive/issues/33)
* [World Models](https://worldmodels.github.io/)
* [1912.01603.pdf](https://arxiv.org/pdf/1912.01603.pdf)
* [Integrating Behavior Cloning and Reinforcement Learning forImproved Performance in Dense and Sparse Reward Environments](https://arxiv.org/pdf/1910.04281.pdf)
* [behavior cloning reinforcement learning - 搜索结果 - 知乎](https://www.zhihu.com/search?q=behavior%20cloning%20reinforcement%20learning&type=content)

## Window 2

* [concatenate - 搜索结果 - 知乎](https://www.zhihu.com/search?type=content&q=concatenate)
* [tf concat - Doge 搜索](https://www.dogedoge.com/results?q=tf+concat)
* [tf.concat与tf.stack的个人理解 - 知乎](https://zhuanlan.zhihu.com/p/37637446)
* [tf.concat()详解_木盏-CSDN博客](https://blog.csdn.net/leviopku/article/details/82380118)

## Window 3

* [tf.io.gfile.exists  |  TensorFlow Core v1.15.0](https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/io/gfile/exists)
* [tf.summary.scalar  |  TensorFlow Core v2.4.1](https://www.tensorflow.org/api_docs/python/tf/summary/scalar)
* [TensorBoard](http://psdz:6006/#scalars)
* [TensorBoard](http://psdz:6007/#scalars)
* [python flag.flag_百度搜索](https://www.baidu.com/s?ie=utf-8&f=8&rsv_bp=1&rsv_idx=1&tn=baidu&wd=%20python%20flag.flag&fenlei=256&oq=flag.flag&rsv_pq=fd25e14b00086fae&rsv_t=41143LU%2BHE1dlk%2FdNBFqTQ63ngKBViylJ%2Bo9wzzEMP6WRAjR0VTLmS0zuic&rqlang=cn&rsv_dl=tb&rsv_btype=t&inputT=2986&rsv_sug3=21&rsv_sug1=3&rsv_sug7=100&rsv_sug2=0&rsv_sug4=3051)
* [python中flag是啥_python中flag什么意思_绝对值.M的博客-CSDN博客_flag在python中是什么意思](https://blog.csdn.net/weixin_35691102/article/details/112935485)
